### PR TITLE
docs: include examples in the docs

### PR DESCRIPTION
**Pull request by Snipy7374**
Hi, don't merge i still need to emphasize some lines (sigh)

### CODE REVIEW

**Comment by Snipy7374** [comment]
Now ready to be merged 🙏🏻

File: docs/source/examples/manager.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    :caption: examples/manager.py - creating a button component
+    :lines: 36-59
+
+
+Customizing your component manager
+----------------------------------
+
+For most use cases, the default implementation of the component manager should suffice. Two methods of interest to customise your managers without having to subclass them are :class:`ComponentManager.as_callback_wrapper` and :class:`ComponentManager.as_error_handler`.
+
+:class:`ComponentManager.as_callback_wrapper` wraps the callbacks of all components registered to that manager along with those of its children. Therefore, if we
```
Comment (Sharp-Eyes) [inline docs/source/examples/manager.rst:57]: These `:class:` directives should be `:meth:` directives. They won't hyperlink until we actually add component managers to the api reference though.

File: docs/source/examples/select.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+A simple example on the use of selects with disnake-ext-components.
+
+For this example, we implement a select menu with double functionality.
+Firstly, the select allows you to select one of three slots. After selecting a
+slot, the select is modified to instead allow you to select a colour. The
+selected slot and colour are then combined to colour the corresponding square.
+
+
+First and foremost, we create a bot as per usual. Since we don't need any
+prefix command capabilities, we opt for an :class:`disnake.ext.commands.InteractionBot`.
```
Comment (Sharp-Eyes) [inline docs/source/examples/select.rst:12]: Could probably make this ``:class:`~disnake.ext.commands.InteractionBot` `` to make it read a little better

**Review by Sharp-Eyes — CHANGES_REQUESTED** [review]
Thanks for the contribution!

I've added a few points that I think could be improved, but it's looking good overall.

Lastly, I'm thinking we could maybe
- Remove comments from the examples on the github repo,
- Add a link to the docs' examples to the repo in `examples/README.md`,
- Keep the extra explanation only in the docs' examples.

What do you think about this?

**Comment by Snipy7374** [comment]
> Thanks for the contribution!
> 
> I've added a few points that I think could be improved, but it's looking good overall.
> 
> Lastly, I'm thinking we could maybe
> - Remove comments from the examples on the github repo,
> - Add a link to the docs' examples to the repo in `examples/README.md`,
> - Keep the extra explanation only in the docs' examples.
> 
> What do you think about this?

Hi, sorry for the late reply.

I'm ok with all the above points, I was already thinking of removing comments when writing all these lines of numbers (it almost seemed like I was crazy giving numbers) also it makes sense that the explanation is on the docs

**Review by Sharp-Eyes — APPROVED** [review]
LGTM! Sorry for the long delay, I finally got back around to working on (and this time around hopefully finishing) the docs.

**Pull request merged**